docker/Dockerfile

```diff
@@ -74,10 +71,101 @@ ARG EXTRA_CMAKE_ARGS=
 RUN git clone ${REPO} ${SOURCE_DIR} \
     && git -C ${SOURCE_DIR} checkout ${COMMIT_OR_TAG} \
     && cmake -H${SOURCE_DIR} -B${BUILD_DIR} -DCMAKE_BUILD_TYPE=${BUILD_TYPE} \
-        -DCMAKE_INSTALL_PREFIX=${INSTALL_DIR} ${EXTRA_CMAKE_ARGS} \
+    -DCMAKE_INSTALL_PREFIX=${INSTALL_DIR} ${EXTRA_CMAKE_ARGS} \
     && cmake --build ${BUILD_DIR} -- -j$(nproc) \
     && cmake --build ${BUILD_DIR} --target install \
-    && rm -rf ${BUILD_DIR}
+    && echo "built files:" && ls -lh ${BUILD_DIR} && echo "installed libs:" && ls -lh ${INSTALL_DIR}/lib \
+    # strip all exes
+    && strip /usr/local/bin/grepperf \
+       /usr/local/bin/zilliqad \
+       /usr/local/bin/genkeypair \
+       /usr/local/bin/signmultisig \
+       /usr/local/bin/verifymultisig \
+       /usr/local/bin/getpub \
+       /usr/local/bin/getaddr \
+       /usr/local/bin/genaccounts \
+       /usr/local/bin/sendcmd \
+       /usr/local/bin/gentxn \
+       /usr/local/bin/restore \
+       /usr/local/bin/gensigninitialds \
+       /usr/local/bin/validateDB \
+       /usr/local/bin/genTxnBodiesFromS3 \
+       /usr/local/bin/getnetworkhistory \
+       /usr/local/bin/isolatedServer \
+       /usr/local/bin/getrewardhistory \
+    #    /usr/local/bin/zilliqa \
+       /usr/local/lib/libSchnorr.so \
+       /usr/local/lib/libethash.so \
+       /usr/local/lib/libNAT.so \
+       /usr/local/lib/libCommon.so \
+       /usr/local/lib/libTrie.so
+
```

Comment:
I suppose below dynamic libraries are not referenced anywhere in the application.
  /usr/local/lib/libSchnorr.so \
  /usr/local/lib/libethash.so \
  /usr/local/lib/libNAT.so \
  /usr/local/lib/libCommon.so \
  /usr/local/lib/libTrie.so
But let it be as even after removing them , we are not saving any space.It's just 5-6 mb's